data/alexandria start.txt

```diff
@@ -0,0 +1,1290 @@
+				accept
+	on visit
+		dialog `You've forgotten your senile friend or his pencils. Perhaps they're in one of your escorts?`
+
+
+
+
+mission "Alexandria Library Card: Second Volume: Step 2"
+	name "Alexandria Library Card"
+	description "Take Dameos Titan to <destination> to search for the second of the five Chal Jub Paqmey."
+	blocked `Daemos complains, "I can't sleep in cooling ducts. I need a bunk and space for 1 ton of cargo."`
+	source Tinker
+	destination Amazon
+	landing
+	passengers 1
+	cargo "old books and a thousand pencils" 1
+	to offer
+		has "Alexandria Library Card: Second Volume: Step 1: done"
+	on offer
+		conversation
+			`You are led to a vast skyscraper overlooking a mountain range. From the glass-walled elevator, you take in the breathtaking view. The elevator stops on the top floor and you see a well-dressed man surrounded by bookshelves. He sitting on a gold-lined leather chair, pulled up to a marble desk. The bookshelves also appear to be hand-carved marble. Whoever this is, he is exceptionally rich.`
+			choice
+				`	(Interrupt him.)`
+			`	The man turns to you and says, "It is always an honor to meet Alexandria Seekers. How can I help you?"`
+			`	Daemos responds, "I seek the Paq cha'DIch, second volume of the Chal Jub Paqmey. I've heard it's your kind of literature. Do you have it?"`
+			`	"I don't have it now, but I can get it for you. Do you have the book, 'A Thousand Poisons of Amazon and How to Conceal Their Taste?' I've been looking for a signed first edition."`
+			`	"Ekhh. A trade. Okay, I'll get you a copy."`
+			`	Daemos whispers to you, "We have to fly to planet Amazon in the Schedar system next. I have the book, but we need to get it signed. I have someone to contact... I'll see you at your ship."`
+			choice
+				`	(Continue.)`
+			`	Daemos meets you at your ship and says, "Our destination is in a dangerous area of Syndicate space. I've arranged some escorts so you don't get me killed."`
+			`	With a disgruntled expression, he asks, "Are you going to pester me with questions again?"`
+				goto "actual questions"
+			label questions
+			`	"Are you going to pester me with questions again?"`
+			label "actual questions"
+			choice
+				`	"What will we do on Amazon?"`
+					goto what
+				`	"Why do we need escorts?"`
+					goto why
+				`	"Sounds good. Let's go."`
+					goto go
+				`	"I don't have time to help you any more."`
+					decline
+			label what
+			`	"I have a first edition of the book that the executive wanted, but we need to get it signed. It's a popular book among the Syndicate, so I always keep a copy or two when I visit their aristocracy. I'm a personal friend of the author, and I know the location of his remote hermitage."`
+				goto questions
+			label why
+			`	Daemos responds, "That area is crawling with pirates and I doubt your ship will survive alone. The Chal Jub Paqmey are critical, so the Library is willing to expend significant resources to obtain it."`
```

Comment:
This was wrong in only two places, and one was intentional. 

The incorrect question from the player is because the player does not know whether the name is plural or not. It is best to assume it is singular, as the object the quest seeks, rather than a group of such objects. Daemos now rudely corrects the player.